### PR TITLE
Release 1.3.0 for develop

### PR DESCRIPTION
**Pull request by tcojean**
This PR pushes the Ginkgo 1.3.0 release for develop. You can find in `CHANGELOG.md` the summary of the additions.

If this is accepted, I will later create a similar PR for the `master` branch.

The documentation will be generated for this branch and will be available on a successful build at:
https://ginkgo-project.github.io/ginkgo/doc/release/1.3.0/index.html

### CODE REVIEW

**Review by upsj — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,68 @@ commits. For a comprehensive list, use the following command:
...
++ For all platforms, cmake 3.9+
++ Linux and MacOS
+  + gcc: 5.3+, 6.3+, 7.3+, all versions after 8.1+
+  + clang: 3.9+
+  + Intel compiler: 2017+
+  + Apple LLVM: 8.0+
+  + CUDA module: CUDA 9.0+
+  + HIP module: ROCm 2.8+
++ Windows
+  + MinGW and CygWin: gcc 5.3+, 6.3+, 7.3+, all versions after 8.1+
```
Comment (yhmtsai) [inline CHANGELOG.md:28]:
```suggestion
  + MinGW and Cygwin: gcc 5.3+, 6.3+, 7.3+, all versions after 8.1+
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,68 @@ commits. For a comprehensive list, use the following command:
...
+  + Microsoft Visual Studio: VS 2017 15.7+
+  + CUDA module: CUDA 9.0+, Microsoft Visual Studio
+  + OpenMP module: MinGW or CygWin.
```
Comment (yhmtsai) [inline CHANGELOG.md:31]:
```suggestion
  + OpenMP module: MinGW or Cygwin.
```

**Review by yhmtsai — APPROVED** [review]
LGTM.
nit: CygWin (in 1.2, 1.3 changelog) -> Cygwin

**Comment by pratikvn** [comment]
Even though the JOSS paper is yet to be published, we could possibly add it to the `CITING.md`

```
@article{Joss2020,
  doi = {10.21105.joss.02260},
  url = {https://doi.org/10.21105/joss.02260},
  year = {2020},
  publisher = {The Open Journal},
  author = {Hartwig Anzt and Terry Cojean and Yen-Chen Chen and Goran Flegar and Fritz G\"{o}bel and Thomas Gr\"{u}tzmacher and Pratik Nayak and Tobias Ribizel and Yu-Hsiang Tsai},
  title = {Ginkgo: A high performance numerical linear algebra library},
  journal = {Journal of Open Source Software}
}
```

We could also wait until the paper is actually published.

**Review by pratikvn — APPROVED** [review]
LGTM!

**Pull request merged**